test/tools/runner/index.js

```diff
@@ -66,6 +66,7 @@ before(function (_done) {
   // );
 
   const options = MONGODB_API_VERSION ? { serverApi: MONGODB_API_VERSION } : {};
+
```

Comment:
this is just a whitespace change, if we get rid of it, we have one less file in the PR

Reply:
Whitespace removed.